receiver/podmanreceiver/metadata.yaml

```diff
@@ -8,3 +8,115 @@ status:
   codeowners:
     active: [rogercoll]
 
+resource_attributes:
+  container.runtime:
+    description: "The runtime of the container. For this receiver, it will always be 'podman'."
+    type: string
+    enabled: true
+  container.id:
+    description: "The ID of the container."
+    type: string
+    enabled: true
+  container.image.name:
+    description: "The name of the image in use by the container."
+    type: string
+    enabled: true
+  container.name:
+    description: "The name of the container."
+    type: string
+    enabled: true
+
+attributes:
+  core:
+    description: "The CPU core number when utilising per-CPU metrics."
+    type: string
+
+metrics:
+  # CPU
+  container.cpu.usage.system:
+    enabled: true
+    description: "System CPU usage."
+    unit: ns
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation_temporality: cumulative
+  container.cpu.usage.total:
+    enabled: true
+    description: "Total CPU time consumed."
+    unit: ns
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation_temporality: cumulative
+  container.cpu.usage.percpu:
+    enabled: true
+    description: "Total CPU time consumed per CPU-core."
+    unit: ns
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation_temporality: cumulative
+    attributes:
+      - core
+  container.cpu.percent:
+    enabled: true
+    description: "Percent of CPU used by the container."
+    unit: 1
+    gauge:
+      value_type: double
+  # Memory
+  container.memory.usage.limit:
+    enabled: true
+    description: "Memory limit of the container."
+    unit: 1
+    gauge:
+      value_type: int
+  container.memory.usage.total:
+    enabled: true
+    description: "Memory usage of the container."
+    unit: 1
+    gauge:
+      value_type: int
+  container.memory.percent:
+    enabled: true
+    description: "Percentage of memory used."
+    unit: 1
+    gauge:
+      value_type: double
+  # Network
+  container.network.io.usage.rx_bytes:
+    enabled: true
+    description: "Bytes received by the container."
+    unit: By
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation_temporality: cumulative
+  container.network.io.usage.tx_bytes:
+    enabled: true
+    description: "Bytes sent by the container."
+    unit: By
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation_temporality: cumulative
+  # BlockIO
+  container.blockio.io_service_bytes_recursive.read:
+    enabled: true
+    description: "Number of bytes transferred from the disk by the container"
+    extended_documentation: "[More docs]i/www.kernel.org/doc/Documentation/cgroup-v1/blkio-controller.txt)."
```

Comment:
It seems that the format of link is not valid. Should it be like this? [More docs](https://www.kernel.org/doc/Documentation/cgroup-v1/blkio-controller.txt).

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30232/commits/df281be2f8410406c032f3c55dfa8e39a931cd44